ntp-proto/src/timetypes.rs

```diff
@@ -125,6 +125,32 @@ impl NtpDuration {
         (((self.duration & 0x0000FFFFFFFF0000) >> 16) as u32).to_be_bytes()
     }
 
+    /// NtpDuration::from_seconds(16.0)
+    pub(crate) const MAXDISP: Self = Self {
+        duration: 68719476736,
+    };
+
+    /// NtpDuration::from_seconds(0.005)
+    #[allow(dead_code)]
+    pub(crate) const MINDISP: Self = Self { duration: 21474836 };
```

Comment:
floating-point arithmetic is not allowed in `const` contexts. So I hardcoded these numbers. Not ideal but I don't see a better way